docs/extras/integrations/text_embedding/onnx_embeddings.ipynb

```diff
@@ -0,0 +1,87 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "719619d3",
+   "metadata": {},
+   "source": [
+    "# ONNX support for any HuggingFace model\n",
```

Comment:
May I suggest changing the title to just "ONNX" or something shorter? This is how it would appear in the docs, which seems "very long" when compared to the other integrations. 

![Screenshot-20230930-22-32-12](https://github.com/langchain-ai/langchain/assets/44113430/01c57e8c-d4ec-4fa7-9c63-7924960930a4)

Reply:
Yes - thank you.